Shared/AuthenticationKeychainInfo.swift

```diff
@@ -35,68 +48,68 @@ public extension KeychainWrapper {
     }
 }
 
-public class AuthenticationKeychainInfo: NSObject, NSCoding {
-    private(set) public var lastPasscodeValidationInterval: NSTimeInterval?
-    private(set) public var passcode: String?
-    private(set) public var requiredPasscodeInterval: PasscodeInterval?
-    private(set) public var lockOutInterval: NSTimeInterval?
-    private(set) public var failedAttempts: Int
-    public var useTouchID: Bool
+open class AuthenticationKeychainInfo: NSObject, NSCoding {
+    fileprivate(set) open var lastPasscodeValidationInterval: TimeInterval?
```

Comment:
`private` here would be better since we have no intention of using this outside this lexical scope.